app/src/main/java/net/squanchy/tweets/model/TweetSpannableText.java

```diff
@@ -0,0 +1,17 @@
+package net.squanchy.tweets.model;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class TweetSpannableText {
```

Comment:
Hm I see this is used _for_ the spannables creation but it isn't a spannable itself, maybe it's worth renaming it